### PR TITLE
fix(fgs): fix fgs trigger support Kafka userName and passWord

### PR DESCRIPTION
**Pull request by jinyangyang222**
**What this PR does / why we need it**:
fix fgs trigger support Kafka userName and passWord
**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
1.add two optional parameters in the schema, 'user' and 'password'
2.add these two parameters to the EvenData request body
```

## PR Checklist

* [ ] Documentation updated.
* [ ] Schema updated.

### CODE REVIEW

File: huaweicloud/services/fgs/resource_huaweicloud_fgs_trigger.go [hunk 1]
```diff
@@ -271,6 +271,16 @@ func kafkaSchemaResource() *schema.Resource {
 				Required: true,
 				ForceNew: true,
 			},
+			"kafka_user": {
```
Comment (Lance52259) [inline huaweicloud/services/fgs/resource_huaweicloud_fgs_trigger.go]: the structure already names `kafka`, the sub-parameters are not recommand start with the `kafka_`.

File: huaweicloud/services/fgs/resource_huaweicloud_fgs_trigger.go [hunk 2]
```diff
@@ -271,6 +271,16 @@ func kafkaSchemaResource() *schema.Resource {
...
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+			},
+			"kafka_password": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
```
Comment (Lance52259) [inline huaweicloud/services/fgs/resource_huaweicloud_fgs_trigger.go]: maybe support a `RequiredWith: []string{"kafka.0.user"}`?

Comment (jinyangyang222) [reply huaweicloud/services/fgs/resource_huaweicloud_fgs_trigger.go]: ok, fixed

**Pull request merged**